resources/documentation/configure_bluebanquise.rst

```diff
@@ -310,14 +310,23 @@ To generate an sha512 password, use the following command (python >3.3):
 
 We need to ensure our management1 node ssh public key is set here.
 
-Get the content of /root/.ssh/id_rsa.pub and add it in this file. At the same time, **remove the ssh key provided here as example**.
+Get the content of /root/.ssh/id_rsa.pub and add it in this file.
+ At the same time, **remove the ssh key provided here as example**.
```

Comment:
blank at start of line, expected?

Reply:
When I see very long lines, I try to split them.
But I don't know if sphinx will add a blank space if I split the line in two. :-|
So I added a blank space.